lib/Component.php

```diff
@@ -67,7 +58,7 @@ public function getScripts() {
         continue;
       }
 
-      $scripts[] = $asset_url . $script;
+      $scripts[] = $this->base_url . $script;
```

Comment:
I can add a check that base_url always ends with a /.  Perhaps a function could be preferrable, to just have one place that does the check above this; in other words checking if the specified URL (script or style) is absolute or relative...  There should be some php way to handle base_url and absolute or relative url spec...